worker/instancemutater/manifold.go

```diff
@@ -5,6 +5,7 @@ package instancemutater
 
```

Comment:
Not one of the issues we're specifically addressing, but on line 81 we have:
```go
return nil, errors.Annotate(err, "cannot start machine instancemutater worker")
```
This is a copy/paste from the machine worker and should refer the the "model" worker. Given that the occurrence of that error raised this bug, we should at least ensure that it is no longer a red herring.

Reply:
k, I fixed the message here.